source/MRCommonPlugins/Basic/MRMoveObjectByMouse.h

```diff
@@ -34,7 +34,9 @@ class MoveObjectByMouse : public StateListenerPlugin<DragStartListener, DragList
     virtual bool onDragStart_( MouseButton btn, int modifiers ) override;
     virtual bool onDrag_( int x, int y ) override;
     virtual bool onDragEnd_( MouseButton btn, int modifiers ) override;
+    virtual void postDraw_() override;
 
+    float menuScaling_;
```

Comment:
You can access it via getViewerInstance().getMenuPlugin()->menu_scaling()
Only test menu before dereferencing